### ISSUE TITLE
Better blank file handling

### ISSUE THREAD

**Issue by tresf** [qzind/tray#346]
We have a client with a `blocked.dat` file that only contains a Carriage Return and Line Feed (`\r\n`)and it prevents QZ Tray from loading.

Here's the backtrace:

```
Could not start tray manager
java.lang.reflect.InvocationTargetException
	at java.awt.EventQueue.invokeAndWait(Unknown Source)
	at java.awt.EventQueue.invokeAndWait(Unknown Source)
	at javax.swing.SwingUtilities.invokeAndWait(Unknown Source)
	at qz.ws.PrintSocketServer.main(Unknown Source)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(Unknown Source)
	at qz.auth.Certificate.existsInFile(Unknown Source)
	at qz.auth.Certificate.isBlocked(Unknown Source)
	at qz.common.TrayManager.addMenuItems(Unknown Source)
	at qz.common.TrayManager.<init>(Unknown Source)
	at qz.ws.PrintSocketServer$1.run(Unknown Source)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
```

What appears to be happening is the logic that loops over each line doesn't check to see if there's data on it and bombs out.

**Comment by bberenz** [comment]
Addressed under 1a12da910ec68eb8e77754a9c00dcc94e0d350c8

**Comment by bberenz** [comment]
Yes... I would swear I thought about that when you first mentioned this issue.. 
7c94891a7634707a2976ef80dcd8492b6dac8609 should cover both
